wiki/History_of_osu!/2009/en.md

```diff
@@ -35,3 +35,128 @@ osu! makes its way to the iPhone from a port to the iPhone by nuudles. The proje
+
+## March
+
+3 updates brought a lot of changes to osu! this month. The first one included filtering options for multiplayer lobbies, as well as a chat option to filter messages if offensive language appeared. If enabled, each chat message containing forbidden entries would be replaced with words defined by a certain list.[^stable-b622]
+
+In the next update being published on 21 March, [kiai time](/wiki/Gameplay/Kiai_time) was added, enabling mappers to emphasise strong parts of songs with star fountains.[^stable-b631]
+
+On 26 March, osu! received support for the TaTaCon wii Taiko drum controller and Wiimote game/menu control support (Catch The Beat only).[^stable-b639][^custom-changelog]
+
+## April
+
+This month, the registered user count exceeded the milestone of 100,000 users.[^user-count]
+
+## May
+
+A change in the kudosu system allowed users to gift kudosu stars to any beatmap, not just their own ones like previously.[^kudosu-change]
+
+Among other changes, osu! got an update this month adding widescreen support and the [Autopilot](/wiki/Gameplay/Game_modifier/Autopilot) mod. Enabling the Autopilot mod would take over the cursor movement and the player would only need to control the tapping.[^stable-b674]
+
+## June
+
+Due to a change to the [Beatmap submission system](/wiki/Beatmapping/Beatmap_submission) users were no longer able to upload any modifications to a beatmap after it was modded. There were cases of beatmaps becoming ranked without the BAT member who ranked the beatmap knowing that it contained [rule-breaking changes](/wiki/Ranking_criteria).[^bss-update]
+
+Before, osu! was only available in English. With the new localisation system coming into place, users became able to translate the game client. They could download a file containing the game's text lines, modify it to their language and ask the file to be integrated into the game client.[^client-localisation][^stable-b695]
+
+## July
+
+On 14 July, osu! news became available as a [RSS feed](https://en.wikipedia.org/wiki/RSS).[^rss-news]
+
+Two updates this month brought a new mouse precision toggle option for low sensitivities as well as new command parameters and shortcuts to simplify [storyboard scripting](/wiki/Storyboard/Scripting) in osu!.[^stable-b753c][^stable-b709]
+
+The database was upgraded with an additional 32 GB SSD to support more storage going forward.[^server-upgrade]
+
+After a first initial test for Taiko online rankings was done in March, ranking statistics became viewable per mode on the osu! website.[^per-mode-ranking][^stable-b622] Prior to that, only the [osu! game mode](/wiki/Game_mode/osu!) had online rankings.
```

Comment:
I do write here that only osu! had online rankings back then, though I haven't found a source if that is actually the case. If someone could verify if this was the case (i.e. CtB had no rankings), I would highly appreciate this.

Reply:
no idea -- this needs a wider audience to take a look at, or peppy's attention